cdap-ui/app/cdap/services/WizardStores/AddNamespace/AddNamespaceStore.js

```diff
@@ -184,32 +186,28 @@ const security = (state = null, action = defaultAction) => {
       return state;
   }
   return Object.assign({}, stateCopy, {
-    __complete: true,
+    __skipped: false,
     __error: action.payload.error || false
   });
 };
 
-const preferences = (state = defaultPreferences, action = defaultAction) => {
+const preferences = (state = null, action = defaultAction) => {
```

Comment:
This cannot be null right? The default state of preferences is `defaultPreferencesState`